library/core/src/num/int_macros.rs

```diff
@@ -3185,7 +3185,8 @@ macro_rules! int_impl {
         /// that code in debug mode will trigger a panic on this case and
         /// optimized code will return
         #[doc = concat!("`", stringify!($SelfT), "::MIN`")]
-        /// without a panic.
+        /// without a panic. If you do not want this behavior consider
```

Comment:
```suggestion
        /// without a panic. If you do not want this behavior, consider
```

Reply:
Sorry @RalfJung. I'll do this in another PR.

Reply:
Done in https://github.com/rust-lang/rust/pull/124246